Driverless AI/machine-learning-experiment-scoring-and-analysis-financial-focus/machine-learning-experiment-scoring-and-analysis-financial-focus.md

```diff
@@ -1039,7 +773,7 @@ Now we are going to run a model diagnostics on the freddie_mac_500_test set. The
 
 5\. Click on the new diagnostics model and a page similar to the one below will appear:
 
-![diagnostics-model-results](assets/diagnostics-model-results.jpg)
+![diagnostics-model-results](assets/diagnostics-model-results.png)
 
 *Things to Note:*
```

Comment:
the number points above can be condense by removing the extra space between each number point